scripts/build_natura_raster.py

```diff
@@ -43,17 +43,22 @@
 import atlite
 import geokit as gk
 
+
 def determine_cutout_xXyY(cutout_name):
-    cutout = atlite.Cutout(cutout_name, cutout_dir="cutouts")
+    cutout = atlite.Cutout(cutout_name, cutout_dir=snakemake.input.cutout_dir)
     x, X, y, Y = cutout.extent
     dx = (X - x) / (cutout.shape[1] - 1)
     dy = (Y - y) / (cutout.shape[0] - 1)
     return [x - dx/2., X + dx/2., y - dy/2., Y + dy/2.]
 
+
 if __name__ == "__main__":
+    if 'snakemake' not in globals():
+        from _helpers import mocksnakemake
+        snakemake = mocksnakemake('build_natura_raster') #has to be enabled
     configure_logging(snakemake)
 
-    cutout_names = np.unique([res['cutout'] for res in snakemake.config['renewable'].values()])
+    cutout_names = {res['cutout'] for res in snakemake.config['renewable'].values()}
```

Comment:
Or this one?

Reply:
You're right, this came on the run while going through the scripts.